test/dependencies.jl

```diff
@@ -122,11 +122,11 @@ end
             dependencies = [
                 Dependency("Zlib_jll")
             ]
-            platform = HostPlatform()
+            platform = HostPlatform(parse(Platform, join(["x86_64", split(Base.BinaryPlatforms.host_triplet(), "-")[2:end]...], "-")))
```

Comment:
What does mean?

Reply:
use exactly what hostplatform gives you, but replace the arch with x86

Reply:
it's hideous, admittedly

Reply:
I think I'm missing what's wrong with keeping `HostPlatform`

Reply:
Bump

Reply:
Yup, you're right.